filament/checkstyle.xml

```diff
@@ -160,5 +160,10 @@
         <module name="AtclauseOrder">
             <property name="tagOrder" value="@param,@return,@throws,@deprecated"/>
         </module>
+
+        <!-- Prevent var for all cases other than new instance creation -->
+        <module name="MatchXpath">
+            <property name="query" value="//VARIABLE_DEF[./TYPE/IDENT[@text='var'] and not(./ASSIGN/EXPR/LITERAL_NEW)]"/>
+        </module>
```

Comment:
![](https://cdn.discordapp.com/emojis/798887423978242078.png)

Reply:
I think you might be right here, not going to be something for the more user facing projects but gradles verboseness makes var useful